README.md

```diff
@@ -1,72 +1,183 @@
-# Microsoft Azure SDK for Go
-[![GoDoc](https://godoc.org/github.com/Azure/azure-sdk-for-go?status.svg)](https://godoc.org/github.com/Azure/azure-sdk-for-go) 
+
+# Azure SDK for Go
+
+[![godoc](https://godoc.org/github.com/Azure/azure-sdk-for-go?status.svg)](https://godoc.org/github.com/Azure/azure-sdk-for-go) 
 [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-go.svg?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-go) 
 [![Go Report Card](https://goreportcard.com/badge/github.com/Azure/azure-sdk-for-go)](https://goreportcard.com/report/github.com/Azure/azure-sdk-for-go)
 
-This is Microsoft Azure's core repository for hosting Go packages which offer a more convenient way of targeting Azure
-REST endpoints. Here, you'll find a mix of code generated by [Autorest](https://github.com/Azure/autorest) and hand
-maintained packages.
+// azure-sdk-for-go provides Go packages for using Azure services
 
-> **NOTE:** This repository is under heavy ongoing development and should be considered a preview. Vendoring your
-dependencies is always a good idea, but it is doubly important if you're consuming this library.
+:exclamation: **NOTE:** This project is in preview and breaking changes are 
+introduced frequently. Therefore, vendoring dependencies is even more
+important than usual. We use [dep](https://github.com/golang/dep).
 
-# Installation
-- If you don't already have it, install [the Go Programming Language](https://golang.org/dl/).
-- Go get the SDK:
+To be notified about updates and changes, subscribe to the [Azure update feed][].
 
-```
+**Install:**
+
+```sh
 $ go get -u github.com/Azure/azure-sdk-for-go/...
 ```
 
-> **IMPORTANT:** We highly suggest vendoring Azure SDK for Go as a dependency. For vendoring dependencies, Azure SDK
-for Go uses [dep](https://github.com/golang/dep).
+If you need to install Go, follow [the official instructions][].
+
+[the official instructions]: https://golang.org/dl/
+
+**Use:**
+
+For complete examples see [Azure-Samples/azure-sdk-for-go-samples][samples_repo].
+
+1. Import a package from the [services][services_dir] directory. See [SERVICES.md][] for
+   a list.
+1. Create and authenticate a client with a `New*Client` func, e.g.
+   `c := compute.NewVirtualMachinesClient(...)`.
+1. Invoke API methods using the client, e.g. `c.CreateOrUpdate(...)`.
+1. Handle responses.
+
+For example, to create a new virtual network (substitute your own values for
```

Comment:
I think that this can be dropped in favor of just keeping in the auth example, but there is the very real concern that using the thin wrappings around the REST data structures can get confusing. Best way to fix that is documentation improvements, though.

Reply:
I think the README is an opportunity to familiarize people with our paradigms early so I'd like to keep a straightforward, short example of client usage here. We could consider combining this with the auth example as long as they can still be read clearly.

Reply:
will leave as is for now but please consider submitting a PR with alternative suggestions!